packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -276,6 +358,31 @@ export abstract class BaseService extends Resource
     };
   }
 
+  /**
+   * Use this function to create all load balancer targets to be registered in this service, add them to
+   * target groups, and attach target groups to listeners accordingly.
+   *
+   * @example
+   *
+   * service.registerContainerTargets(
+   *   {
+   *     containerTarget: {
+   *       containerName: 'web',
+   *       containerPort: 80,
+   *     },
+   *     targetGroupId: 'ECS',
+   *     listener: ecs.ListenerConfig.applicationListenerConfig(listener, {
+   *       protocol: elbv2.ApplicationProtocol.HTTPS
+   *     }),
+   *   },
+   * )
+   */
+  public registerContainerTargets(...targets: EcsTarget[]) {
```

Comment:
Are you sure that, if you're coming at this cold (and don't have all the state that you have in your head about this change), that the name `service.registerContainerTargets` is clear enough that it pertains to load balancing? (And not, for example, service discovery?)

For myself, I would be wondering *what* I'm registering into?

Reply:
It is hard to find a name that can be really clear enough to figure out what exactly this method does by seeing the name alone. However, I agree the naming can be improved to make it more explicit.